Packages/Status/Sources/Status/Editor/StatusEditorViewModel.swift

```diff
@@ -72,9 +82,10 @@ public class StatusEditorViewModel: ObservableObject {
   private var currentSuggestionRange: NSRange?
 
   private var embeddedStatusURL: URL? {
-    return URL(string: embeddedStatus?.reblog?.url ?? embeddedStatus?.url ?? "")
+    URL(string: embeddedStatus?.reblog?.url ?? embeddedStatus?.url ?? "")
   }
 
+  private var mentionString: String?
```

Comment:
This is never set or am I missing something?

Reply:
It gets set on line 188.